browser/ui/webui/settings/brave_settings_localized_strings_provider.cc

```diff
@@ -948,7 +956,11 @@ void BraveAddCommonStrings(content::WebUIDataSource* html_source,
       confirmation_phrase);
   html_source->AddString("walletResetTransactionInfoConfirmation",
                          reset_tx_confirmation_text);
-
+  auto reset_zcash_sync_state_confirmation_text = l10n_util::GetStringFUTF16(
+      IDS_SETTINGS_WALLET_RESET_ZCASH_SYNC_STATE_CONFIRMATION,
+      confirmation_phrase);
+  html_source->AddString("resetZCashSyncStateConfirmation",
+                         reset_zcash_sync_state_confirmation_text);
   html_source->AddString(
       "braveLeoAssistantInputDefaultContextSize",
       base::NumberToString16(ai_chat::kDefaultCustomModelContextSize));
```

Comment:
Do we need these temp variables like reset_zcash_sync_state_confirmation_text or reset_tx_confirmation_text, etc. Can we have them inlined?

Reply:
Removed